srml/staking/src/lib.rs

```diff
@@ -1322,6 +1386,46 @@ impl<T: Trait> OnFreeBalanceZero<T::AccountId> for Module<T> {
 	}
 }
 
+/// Add reward points to block authors:
+/// * 20 points to the block producer for producing a (non-uncle) block in the relay chain,
+/// * 2 points to the block producer for each reference to a previously unreferenced uncle, and
+/// * 1 point to the producer of each referenced uncle block.
+// TODO TODO: either we make authorship bound on trait either we make authorship dependency optinal
```

Comment:
this can be done in another PR, just a new feature and make tests and mock using authorship behind the feature